webapp/src/components/molecules/Login.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { Button } from 'primereact/button';
+import { Dropdown } from 'primereact/dropdown';
+import { Link } from 'react-router-dom';
+import 'primereact/resources/themes/saga-blue/theme.css';
+import 'primereact/resources/primereact.min.css';
+
+const loggedInDropdown = [
+  { label: 'View profile', value: '/profile' },
+  { label: 'Logout', value: '/logout' },
+];
+
+export interface LoginProps {
+  user: {
+    name?: string;
+    isLoggedIn: boolean;
+  };
+}
+
+export default function Login(props: LoginProps) {
```

Comment:
I really like this, could you maybe rename both the function and the file to something like `UserDropown` `LoginButton` `UserLoginManager` or something a little more descriptive?